website/src/views/covid.ts

```diff
@@ -75,6 +75,15 @@ export class CovidAnalyzeSingleVariantView extends CovidConstants implements Vie
     public readonly pathname = `/${pathFragment}/single-variant` as const;
     public readonly label = 'Single variant';
     public readonly labelLong = 'Analyze a single variant';
+    public readonly title = `${this.label} | ${organismConfig[this.organism].label} | GenSpectrum`;
+    public readonly breadcrumbMenuEntries = getViewBreadcrumbEntries(
+        {
+            name: this.label,
+            href: this.pathname,
+        },
+        organismConfig[this.organism].label,
+    );
```

Comment:
This is the same code in every view. Can we extract this somehow? Inheriting a method from some abstract parent? Having a method `view: View => BreadcrumbElement[]` somewhere else?

Reply:
Is discussed: We now pass a view to the `getViewBreadcrumbEntries` function and use it in the `OrganismPageLayout` directly.